### PR TITLE
Use docs API to render setup instructions

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Uses the docs API (`www.highlight.io/api/quickstart`) to pull in content for the setup page.

## How did you test this change?

Local click test and PR preview.

## Are there any deployment considerations?

No backend changes, just need to make sure people are finding what they need for integrating Highlight.

### CODE REVIEW

File: frontend/src/pages/Setup/SetupPage.tsx [hunk 1]
```diff
@@ -134,11 +184,21 @@ const SetupPage = ({ integrated }: { integrated: boolean }) => {
 		project_id,
 	])
 
+	useEffect(() => {
+		fetch(`https://www.highlight.io/api/quickstart`)
+			.then((res) => res.json())
+			.then((docs) => {
+				setDocs(docs)
+				verifyDocsMapping(docs)
```
Comment (ccschmitz) [inline frontend/src/pages/Setup/SetupPage.tsx:192]: Only run in dev

File: frontend/src/pages/Setup/SetupPage.tsx [hunk 2]
```diff
@@ -43,44 +36,101 @@ interface SetupStep {
...
+	chi: 'Chi',
+	mux: 'Mux',
+	gin: 'Gin',
+	apollo: 'Apollo',
+	cloudflare: 'Cloudflare',
+	express: 'Express',
+	firebase: 'Firebase',
+	nodejs: 'Node.js',
+	trpc: 'tRPC',
+} as const
```
Comment (Vadman97) [inline frontend/src/pages/Setup/SetupPage.tsx:86]: can leverage new key - https://github.com/highlight/highlight.io/pull/420

https://github.com/highlight/highlight.io/blob/e85c06640e5648b41f7525edf1544de34995d942/components/QuickstartContent/QuickstartContent.tsx#L40

**Review by Vadman97 — APPROVED** [review]
coming together nicely

File: frontend/src/pages/Setup/SetupPage.tsx (hunk 2 above)
Comment (ccschmitz) [reply frontend/src/pages/Setup/SetupPage.tsx]: Updated in https://github.com/highlight/highlight/pull/4532/commits/ab6a840778605c25cf9ebb86c4c5a10a7b3fb703

**Pull request merged**